vaas/vaas/cluster/cluster.py

```diff
@@ -11,40 +11,31 @@
 from django.conf import settings
 
 from vaas.api.client import VarnishApi, VarnishApiReadException
+from vaas.cluster.exceptions import VclLoadException
 from vaas.cluster.models import VarnishServer, LogicalCluster
+from vaas.metrics.handler import metrics
 from vaas.vcl.loader import VclLoader, VclStatus
 from vaas.vcl.renderer import VclRenderer, VclRendererInput, init_processing, collect_processing, Vcl
-from vaas.cluster.exceptions import VclLoadException
-from statsd.defaults.django import statsd
-from datetime import datetime
 
 
 @app.task(bind=True, soft_time_limit=settings.CELERY_TASK_SOFT_TIME_LIMIT_SECONDS)
 def load_vcl_task(self, emmit_time, cluster_ids):
-    emmit_time_aware = timezone.make_aware(datetime.strptime(emmit_time, "%Y-%m-%dT%H:%M:%S.%fZ"),
-                                           timezone=timezone.utc)
-    if settings.STATSD_ENABLE:
-        queue_time_from_order_to_execute_task = timezone.now() - emmit_time_aware
-        statsd.timing('queue_time_from_order_to_execute_task', queue_time_from_order_to_execute_task)
+    emmit_time_aware = time.perf_counter()
+    metrics.time('queue_time_from_order_to_execute_task', time.perf_counter() - emmit_time_aware)
 
     start_processing_time = timezone.now()
     clusters = LogicalCluster.objects.filter(
         pk__in=cluster_ids, reload_timestamp__lte=emmit_time
     ).prefetch_related('varnishserver_set')
     if len(clusters) > 0:
         varnish_cluster_load_vcl = VarnishCluster().load_vcl(start_processing_time, clusters)
-        if settings.STATSD_ENABLE:
-            statsd.gauge('events_with_change', 1)
-            total_time_of_processing_vcl_task_with_change = timezone.now() - emmit_time_aware
-            statsd.timing('total_time_of_processing_vcl_task_with_change',
-                          total_time_of_processing_vcl_task_with_change)
+        metrics.gauge('events_with_change', 1)
```

Comment:
For such metric I think we should use gauge with increment, example is here: https://github.com/prometheus/client_python#gauge
For statsd it is also possible: https://github.com/statsd/statsd/blob/master/docs/metric_types.md#gauges

I would like to change it, because currently charts with data from this metrics are completely not usefully, example here:
https://metrics.allegrogroup.com/d/TYzEYxoGk/vaas-app-metrics-dev-test-prod?orgId=1&var-Source=Skystats%20prod&var-runtime=k8s_dc&from=now-12h&to=now&viewPanel=47

Maybe we can have two methods? :) What do you think about it?
`guage_set()`
`guage_inc()`

Reply:
It sounds like "Counter". I also noticed that when I was developing this, but I left it for consistency.

Reply:
Ok, for quick rethinking I will add Counter to MetricHandler and migrate those gauges to counters.